app/src/main/res/values/strings.xml

```diff
@@ -646,5 +646,6 @@
     </string-array>
     <string name="app_info">App Info</string>
     <string name="login_failed">Login Failed, Please Try Again Later.</string>
-
+    <string name="could_not_update_password_error">Could not update password. Please try again later.</string>
```

Comment:
Its a long string, keep it concise. How about `We were unable to update password` ?